Kryptonite/Silo.swift

```diff
@@ -111,16 +109,18 @@ class Silo {
         case .noOp:
             return
             
-        case .hosts,
-             .ssh where Policy.needsUserApproval(for: session, and: request.body),
-             .git where Policy.needsUserApproval(for: session, and: request.body):
-            
+        default: break
```

Comment:
can we keep this as each case enumerated for when we add new cases?

Reply:
👍 these are special cases only, but it doesn't hurt to add the new cases.